### ISSUE TITLE
Video prior appears to be trained with ground-truth actions, but the paper describes it as action-free

### ISSUE THREAD

**Issue by jhq1234** [world-action-verifier/wav_minigrid#9]
Thanks for releasing the code.

The paper states the subgoal generator / video prior `g_φ(s_{t+1}|s_t)` is trained without action labels
(Appendix E.2.2: *"28,000 transitions is used as an unlabeled pre-training set to train the video model
**without action annotations**"*), and this is central to the first asymmetry in the abstract.

In the released MiniGrid code, however, `train_vp.py` passes ground-truth actions unconditionally:

```python
# exps/train/train_vp.py
pred_out = model(inputs, next_obs_inputs=next_inputs, mode='posterior',
                 gt_actions=batch['action'].to(DEVICE))
```

and `WorldModel.forward` then indexes the VQ codebook by the true action:

```python
# src/wav_minigrid/models/wm.py
if gt_actions is not None:
    indices = gt_actions.long()
    z_q = self.vq_layer._embedding(indices)   # supervised VQ
    vq_loss = torch.tensor(0.0)
outputs['target_indices'] = indices           # == gt_actions
```

so in `compute_loss_vp`, `loss_prior = CE(prior_logits, target_indices)` is a cross-entropy against the
ground-truth action, applied with `PRIOR_WEIGHT = 5.0`. (`AUX_ACTION_WEIGHT = 0` does not remove this —
`loss_aux_action` computes the identical CE, so setting it to 0 only avoids double counting.)

`data/MiniGrid-Empty-Interact-6x6-o3-v0_video_pretraing.npz` does contain real action labels
(7 classes, counts `[4040, 4052, 2683, 4016, 3798, 4044, 2367]`), so these are not placeholders.

I realize the `else` branch (`gt_actions=None`) implements an unsupervised VQ latent-action path, which
looks like the intended action-free formulation. Two questions:

1. Were the paper's MiniGrid results produced with the supervised path shown above, or with the
   `gt_actions=None` path?
2. If the latter, could you share the corresponding config? With the released
   `VIDEO_TRAINING["VQ_LOSS_WEIGHT"] = 0`, the action-free path cannot train the codebook — because of
   the straight-through estimator (`quantized = inputs + (quantized - inputs).detach()`), `vq_loss` is
   the only gradient path to `vq_layer._embedding`.

For reference, I ran the `gt_actions=None` path with `VQ_LOSS_WEIGHT` swept over {0.1, 0.25, 1, 4, 8, 16};
values below ~1 diverge, and 8 was best. Feeding the resulting prior through
`train_wm.py` → `wm_active_learning.py` (single seed, SEED=48) gave a noticeably different picture from
the shipped checkpoints, so I would like to make sure I am reproducing the intended setup.

Separately: `wav_robot` does not appear to contain a subgoal generator at all — `score_idm` compares the
IDM-implied rollout against the ground-truth next latent from an already-collected episode. Is the
video-prior component planned for that release?
